german/paddeln/paddeln.md

```diff
@@ -0,0 +1,15 @@
+Mühsam komme Sie mit dem Ordner als Paddel voran. Nach und nach weicht der Ordner mehr
+und mehr durch. Bereits nach fünf Minuten ist die Pappe völlig durchnässt und der Ordner
+löst sich auf. Das Boot gleitet langsam aus. Die Sonne brennt Ihnen auf den Kopf.
+
+Was machen Sie nun?
+
+[Laut um Hilfe schreien](../Hilfeschrei/Hilfeschrei.md)?
+
+[Aus dem Boot springen und nach Norden schwimmen?](../schwimmen/schwimmen.md)
+
+[Aus dem Traum aufwachen und Oktoberfest besuchen](../Oktoberfest/oktoberfest.md)
+
+[Die Stille geniessen und nachdenken](entspannen/entspannen.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.